tests/test_snowflake_common.py

```diff
@@ -212,8 +213,9 @@ def test_get_slice_with_limit_without_offset_not_enough_data(
 def test_get_slice_with_limit_with_offset(result, execute_query, connect, snowflake_datasource):
     ds: DataSlice = SnowflakeCommon().get_slice(connect, snowflake_datasource, offset=5, limit=5)
     assert result.call_count == 3
+    # FIXME: What ?? There should be only 5 rows
```

Comment:
Yeah, that's probably because we're not mocking the right value at line **210**, it should be `data_result_5` instead of `data_result_all`

Reply:
Good catch :+1: